examples/test_plan/test_plans.py

```diff
@@ -69,7 +69,7 @@
 
 # Query test plan using id.
 query_test_plans_request = QueryTestPlansRequest(
-    skip=0, take=1, descending=False, returnCount=False
+    take=1, descending=False, return_count=False
```

Comment:
Note: `QueryTestPlansRequest` doesn't have a `skip` property (though it does have a `continuation_token` property). Since it was passing `skip=0`, I'm assuming it would have no effect whether the service supports a `skip` parameter or not.